### ISSUE TITLE
Incorrect parentage GO:0010515    negative regulation of induction of conjugation with cellular fusion

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#20936]
GO:0010515    negative regulation of induction of conjugation with cellular fusion

is a descendant of
GO:0031139    positive regulation of conjugation with cellular fusion

so, all our negative regulators of conjugation are annotated to "positive regulation"

(I don't really understand the difference between  positive regulation, and induction in the context of regulating conjugation?)

CC @Antonialock

**Comment by ValWood** [comment]
I put this at high priority because it's very misleading... I'm trying to approve a conjugation/sexual differention  paper and  I'm way out of my comfort zone.... There seem to be  way too many ways to say the same thing...

**Comment by pgaudet** [comment]
Right, we have 'negative regulation of positive regulation' --- very confusing indeed. 

<img width="889" alt="image" src="https://user-images.githubusercontent.com/4782928/108980825-64964200-768c-11eb-873c-7e9c422ae367.png">

**Comment by pgaudet** [comment]
Are you sure 'GO:0010515 negative regulation of induction of conjugation with cellular fusion' is a process? All annotations are IMP by pombase. The first paper I look at, PMID:1863602, is a mutant that fails to induce conjugation. 

Same for PMID:10792724 - it looks like the signaling required for induction is disrupted; but not that the genes mutated are negatively regulating induction of conjugation.

One possibility would be to merge 
GO:0010515 negative regulation of induction of conjugation with cellular fusion 
into
GO:0031138 negative regulation of conjugation with cellular fusion
(but that may not even correctly capture the data)

Can you please have a look ?

**Comment by ValWood** [comment]
I am new to annotating this area, @Antonialock  used to do this as it is her area of expertise.

I do know that the cAMP pathway specifically inhibits conjugation:

![F1 large](https://user-images.githubusercontent.com/7359272/108992527-5565c300-7691-11eb-81a6-11d5897ca012.jpg)

Biologically it is correct, and the annotation is correct-  cAMP  signalling inhibits conjugation.
the mutants " They conjugate and sporulate under conditions that normally inhibit wild-type strains." **because the negative control is lost**. Overexpression fails to induce conjugation.

But I don't know how we should capture this in GO. It seems that what is regulated  is the 'switch' for sexual differentiation, and that 'conjugation is the first step after the switch.

I'm struggling to separate
GO:0010514    induction of conjugation with cellular fusion
"GO:0032005 signal transduction involved in positive regulation of conjugation with cellular fusion"
"GO:0110044    regulation of cell cycle switching, mitotic to meiotic cell cycle"
GO:0110046    signal transduction involved in cell cycle switching, mitotic to meiotic cell cycle
GO:1900237    positive regulation of induction of conjugation with cellular fusion


 PMID:10792724  seems  to  describing the mechanism  to turn off the signal by degrading the pheromone.  So this looks like negative regulation too?
But I think this is  maybe should only be "negative regulation of pheromone signalling" (seems to be to allow cells to return to mitotic cycle if conjugation did not take place).

Again, I defer to @Antonialock as this is her PhD supervisors work!

To me the merges would make sense, but @Antonialock  would need to confirm. I'm way out of my comfort zone. I'm curating a  checking  sexual differentiation paper right now and I'm on day two I'm having to do so much reading to  get my head around it!

**Comment by pgaudet** [comment]
Hi @Antonialock - have you had a chance to look at this one?

**Comment by pgaudet** [comment]
I'm going through the high priority - @Antonialock if you find a bit of time to help with this it'd be great.

**Comment by Antonialock** [comment]
I agree there are too many terms in this area. I did most annotation right when I started so many are old, I remember it was confusing understanding the diffference between them. I think a lot of terms could be purged. I'll try and take a look.
